src/transformers/models/bert/modeling_tf_bert.py

```diff
@@ -667,6 +827,29 @@ def call(
         ten_thousand_cst = tf.constant(-10000.0, dtype=embedding_output.dtype)
         extended_attention_mask = tf.multiply(tf.subtract(one_cst, extended_attention_mask), ten_thousand_cst)
 
+        # Copied from `modeling_tf_t5.py` with -1e9 -> -10000
```

Comment:
Using -10000 makes sense to me - I'm guessing you made this change to avoid float16 underflow? We should possibly change it in the original T5 code too, since e^-10000 is enough to get to 0 even for 64-bit floats, let alone 32 or 16-bit.

Reply:
I change to -10000 simply because this is the value used in TFBert (before this PR), I think.

Reply:
I think -10000 is much better, because -1e9 will end up as -inf in float16. That might still work okay in the attention mask, but I'd prefer to avoid it.